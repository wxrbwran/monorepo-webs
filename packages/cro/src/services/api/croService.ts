import http from '@/services/http';
import { CommonData } from 'typings/global';

export default {
  // 我的团队 查询待添加的医生列表, 多选医生作为好友
  fetchBaseRoles(data: CommonData): Promise<any> {
    return http.get('base/roles', { data });
  },

  // 我的团队 查询好友列表
  fetchDoctorFriends(data: CommonData): Promise<any> {
    return http.get('doctor/friends', { data });
  },

  // 我的团队 查询好友列表
  putTeamMembers(data: CommonData): Promise<any> {
    return http.put('doctor/team/members', { data });
  },

  // 我的团队 查询好友列表
  getTeams(data: CommonData): Promise<any> {
    return http.get(`doctor/teams?data=${JSON.stringify(data)}`);
  },




};