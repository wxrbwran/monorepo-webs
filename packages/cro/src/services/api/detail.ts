import http from '@/services/http';

export default {
  fetchMedicines(data: any): Promise<any> {
    return http.get(`medicine/name?data=${JSON.stringify(data)}`);
  },
  fetchDiagnosis(data: any): Promise<any> {
    return http.get(`disease?data=${JSON.stringify(data)}`);
  },
  // fetchUploadInfo(file: object): Promise<any> {
  //     return http.get(`clinical/form/address?fileName=${file.name}`);
  // },
  fetchUploadInfo(data?: any): Promise<any> {
    return http.get(`cro/file/address?projectSid=${data.projectSid}&fileName=${data.fileName}`);
  },
  fetchTreatment(data: any): Promise<any> {
    return http.get(`treatment?data=${JSON.stringify(data)}`);
  },
  // 获取项目详情
  getCroProjectInfo(value: string): Promise<any> {
    return http.get(`research/project/detail/${value}`);
  },
  // 添加文件
  addProjectFile(data?: any): Promise<any> {
    return http.post('research/project/file', { data });
  },
  // 文件列表
  getProjectFileList(data?: any): Promise<any> {
    return http.get(`research/project/file?data=${JSON.stringify(data)}`);
  },
  // 删除文件
  deleteProfileFile(fileId: string): Promise<any> {
    return http.delete(`research/project/file/${fileId}`);
  },
  //更新项目信息
  updateCroProject(data?: any): Promise<any> {
    return http.patch('research/project', { data });
  },
  //添加加入、排除标准
  addStandard(data?: any): Promise<any> {
    return http.post('cro/project/standard', { data });
  },
  //查看加入、排除标准
  getStandard(value: string): Promise<any> {
    return http.get(`cro/project/standard/${value}`);
  },
  //获取客观量表（检查提醒）
  getObjectiveList(data?: any): Promise<any> {
    return http.get(`research/objective/examination/scale/${data}`);
  },
  //预览文件
  getFileInfo(fileId: any): Promise<any> {
    return http.get(`research/project/file/info/${fileId}`);
  },
};
