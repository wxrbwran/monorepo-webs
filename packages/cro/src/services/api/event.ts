import http from '@/services/http';

export default {
  // 添加项目终点事件
  addEndEvent(data?: any): Promise<any> {
    return http.post('research/project/end_event',{data});
  },
  // 查看项目终点事件
  fetchEndEvent(projectSid: any): Promise < any > {
    return http.get(`research/end_event/detail/${projectSid}`);
  },
  // 修改项目终点事件
  updateEndEvent(data?: any): Promise < any > {
    return http.patch('research/project/end_event', { data });
  },
  // 查询项目中所有研究者
  fetchResearchers(projectNsId: any): Promise < any > {
    return http.get(`research/project/researchers/${projectNsId}`);
  },
  // 查询项目终点事件统计信息
  fetchEventCountInfo(data?: any): Promise < any > {
    return http.get(`research/project/event_statistics?data=${JSON.stringify(data)}`);
  },
}
