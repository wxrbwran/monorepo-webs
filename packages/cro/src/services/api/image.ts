import http from '@/services/http';

export default {
  // 获取化验单检查单统计数据接口new
  // fetchImageCountNew(data: any): Promise<any> {
  //   return http.get('image/count/last', { data });
  // },
  // 获取具体类型化验单检查单图片
  // fetchImageDetail(data: any): Promise<any> {
  //   return http.get('image/detail', { data });
  // },
  // // 获取具体类型化验单检查单图片V1版接口
  // fetchImageDetailV1(data: any): Promise<any> {

  //   return http.get('image/detail/v1', { data });
  // },
  // // 修改图片旋转角度
  // patchImageDegree(data: any): Promise<any> {
  //   return http.patch('image/detail', { data });
  // },
  // // 获取自定义指标历史中所有的图片分类
  // fetchImageTypeTabs(data: any): Promise<any> {
  //   return http.get('image/type/last', { data });
  // },
  // // 获取某一项图片分类的所有指标详情
  // fetchImageIndexHistory(data: any): Promise<any> {
  //   return http.get('image/index/history/last', { data });
  // },
  // 获取图片结构化数据指标
  // fetchImageIndex(data: any): Promise<any> {

  //   return http.get('nurse/image/index', { data });
  // },
  // // 保存图片结构化数据指标
  // putImageIndex(data: any): Promise<any> {
  //   return http.put('nurse/image/index', { data });
  // },

  // // 获取图片结构化接口
  // fetchImageIndexCustom(data: any): Promise<any> {
  //   return http.get('nurse/image/index/custom', { data });
  // },
  // // 修改图片结构化数据指标
  // putImageIndexCustom(data: any): Promise<any> {
  //   return http.put('nurse/image/index/custom', { data });
  // },

  // // 获取所有子分类，即检查部位
  // fetchImageSubtypes(): Promise<any> {
  //   return http.get('image/subtypes');
  // },
  // // 搜索图片大分类或者指标
  // fetchImageIndexInfo(data: any): Promise<any> {
  //   return http.get('image/index/info', { data });
  // },
  // // 结构化
  // // 获取图片信息及结构化数据
  // putImage(data: any): Promise<any> {
  //   return http.put('image', { data });
  // },
  fetchImageIndexes(data: any): Promise<any> {
    return http.get(`image/indexes?data=${JSON.stringify(data)}`);
    // return http.get('image/indexes', { data });
  },
  // // 保存图片信息及结构化数据
  // putImageImageIndexes(data: any): Promise<any> {
  //   return http.put('image/indexes', { data });
  // },
  // 获取检查单、其它单据
  fetchImageJcdAndOther(data: any): Promise<any> {
    // return http.get('image/jcd', { data });
    return http.get(`image/jcd?data=${JSON.stringify(data)}`);
  },
  // // 新增检查单、其它单据
  // putImageJcdAndOther(data: any): Promise<any> {
  //   return http.put('image/jcd', { data });
  // },
  // // 获取检查单问题模板
  // fetchImageTopicTemplate(data: any): Promise<any> {
  //   return http.get('image/template', { data });
  // },
  // // 新增检查单问题模板   11.18新增系统/医生检查单
  // putImageTopicTemplate(data: any): Promise<any> {
  //   return http.put('image/template', { data });
  // },
  // // 新增/修改/删除检查单问题
  // postImageTemplate(data: any): Promise<any> {
  //   return http.post('image/template', { data });
  // },
  // // 修改检查单模板名称
  // patchImageTemplateName(data: any): Promise<any> {
  //   return http.patch('image/template', { data });
  // },
  // // 获取检查单检查部位或检查方法
  // fetchImageTemplatePart(): Promise<any> {
  //   return http.get('image/template/part');
  // },
  // // 根据检查部位或检查方法获取检查名称
  // fetchImageTemplateName(data: any): Promise<any> {
  //   return http.get('image/template/name', { data });
  // },
  // 结构化
};
