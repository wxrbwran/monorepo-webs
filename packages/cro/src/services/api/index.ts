export { default as auth } from './auth';
export { default as detail } from './detail';
export { default as user } from './user';
export { default as subjective } from './subjective';
export { default as project } from './project';
export { default as patientManage } from './patientManage';
export { default as report } from './report';
export { default as query } from './query';
export { default as template } from './template';
export { default as research } from './research';
export { default as base } from './base';
export { default as event } from './event';
export { default as service } from './croService';
export { default as image } from './image';