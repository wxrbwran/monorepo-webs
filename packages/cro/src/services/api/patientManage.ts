
import http from '@/services/http';

export default {
  // 获取全部患者
  getPatientList(data: any): Promise<any> {
    return http.get(`research/teams/patient?data=${JSON.stringify(data)}`);
  },
  // 获取受试者列表
  getTestPatients(data: any): Promise<any> {
    return http.get(`research/teams/project/patient?data=${JSON.stringify(data)}`);
  },
  // 全部患者列表医生手动打钩 勾选患者
  postCheckPatient(data: any): Promise<any> {
    return http.post('research/patient/checked', { data });
  },

  // 添加或修改 受试者在项目的给药时间或停止此项目用药时间
  patchPatientMedTimeStatus(data: any): Promise<any> {
    return http.patch('research/project/patient/med_time', { data });
  },
  // research/project/patient/med_time


  //全部患者列表医生去除打钩
  // patchDelCheckPatient(data: any): Promise<any> {
  //   return http.patch('cro/project/check/patient/type', { data });
  // },
  // 创建试验小组
  postAddGroup(data: any): Promise<any> {
    return http.post('research/project/group', { data });
  },
  // 获取小组列表
  getGroupList(projectNsId: any): Promise<any> {
    return http.get(`research/group/${projectNsId}`);
  },
  // 获取入组排除标准列表
  getStandard(projectSid: any): Promise<any> {
    return http.get(`cro/project/standard/${projectSid}`);
  },
  // 查看项目文件列表
  getProjectFile(projectSid: any, data: any): Promise<any> {
    return http.get(`cro/project/file/${projectSid}`, { data });
  },
  // 发送试验项目书、知情同意书 发送项目邀请
  postSendFile(data: any): Promise<any> {
    return http.post('research/project/invite', { data });
  },
  // 私下已签署，直接添加患者为受试者
  postJoinProject(data: any): Promise<any> {
    return http.post('research/project/join/by_doctor', { data });
  },
  // 停止参与试验 更改受试者状态
  patchPatientStatus(data: any): Promise<any> {
    return http.patch('research/project/patient', { data });
  },
  // 向小组添加患者 加入实验分组
  postGroupPatient(data: any): Promise<any> {
    return http.post('research/group/patient', { data });
  },
  // 查询小组内患者
  getGroupPatient(data: any): Promise<any> {
    return http.get(`research/group/patient?data=${JSON.stringify(data)}`);
  },
  //修改试验小组名
  modifyGroup(data: any): Promise<any> {
    return http.patch('research/project/group', { data });
  },
};
