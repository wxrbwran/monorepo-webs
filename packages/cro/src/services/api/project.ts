import http from '@/services/http';

export default {
  // 创建项目
  postCroProject(data: any): Promise<any> {
    return http.post('research/project', { data });
  },
  // 获取项目列表
  getProjectList(data: any): Promise<any> {
    // return http.get(`research/all/projects?data=${JSON.stringify(data)}`);
    // return http.get(`research/all/projects?doctorId=${1381}`);
    return http.get(`research/all/projects`);
  },
  // 获取患者统计图表
  getPatientStatistic(): Promise<any> {
    return http.get(`research/patient/statistics`);
  },
  // 获取项目统计图表
  getPStatistic(): Promise<any> {
    return http.get(`research/project/statistics`);
  },
  //删除项目
  delProject(data: any): Promise<any> {
    return http.delete(`research/project/${data}`);
  },
};
