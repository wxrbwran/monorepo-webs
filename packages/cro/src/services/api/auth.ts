import { auth, ajax } from '@/services/auth';
import qs from 'qs';

export default {
  // 登录
  token(formData: CommonData): Promise<any> {
    return auth.post('token', {
      data: qs.stringify(formData)
    });
  },
};
