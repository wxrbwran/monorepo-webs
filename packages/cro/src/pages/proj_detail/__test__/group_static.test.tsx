import * as React from 'react'
import { render, RenderResult, fireEvent, waitFor, act } from '@testing-library/react';
import GroupStatic from '../group_static';
import * as api from '@/services/api';

let wrapper: RenderResult;
const props = {
  projectSid: '5beKM8'
};

jest.mock("@/services/api", () => {
  return {
    detail: {
      getStandard: jest.fn(() => {
        return Promise.resolve({
          infos: [], status: 'success'
        });
      }),
    }
  };
});

describe('项目详情：纳入标准、排除标准', () => {
  beforeEach(() => {
    wrapper = render(<GroupStatic {...props} />)
  })
  it('匹配快照', () => {
    expect(wrapper).toMatchSnapshot();
  })
  it('未设置过纳入标准，显示添加按钮', () => {
    expect(wrapper.queryAllByText('添加条件').length).toBe(2);
  })
  it('点击添加按钮，增加一行选项;点击删除icon，删除一行选项', async() => {
    // add
    const addBtn = wrapper.getByTestId('addJoin');
    const delBtn = wrapper.getByTestId('delJoin');
    const selector = wrapper.container.getElementsByClassName('ant-select-selector');
    expect(selector.length).toBe(2);
    fireEvent.click(addBtn);
    expect(selector.length).toBe(3);
    fireEvent.click(delBtn);
    expect(selector.length).toBe(2);
  })
  it('点击提交按钮，提示提交后不可修改', () => {
    const submit = wrapper.getByTestId('submit');
    fireEvent.click(submit);
    expect(wrapper.getByText('提交之后不可修改，您确定提交吗？')).toBeInTheDocument();
  })
  it('点击确认提交按钮，标准为空，提交失败，提示不能为空', () => {
    const submit = wrapper.getByTestId('submit');
    fireEvent.click(submit);
    const confirmSubmit = wrapper.getByText('确 认');
    fireEvent.click(confirmSubmit);
    expect(wrapper.getByText('纳入标准排除标准不能同时为空')).toBeInTheDocument();
  })
  // it('纳入标准选择年龄后，显示范围输入框', async() => {
  //   const select = wrapper.container.querySelector('.ant-select-selection-search-input');
  //   select && fireEvent.change(select, {
  //     target: { value: 'AGE' }
  //   })
  //   wrapper.debug();
  //   await waitFor(() => {
  //     const dropdown = wrapper.getByText('范围：');
  //     expect(dropdown).toBeInTheDocument();
  //   })
  // })
})
