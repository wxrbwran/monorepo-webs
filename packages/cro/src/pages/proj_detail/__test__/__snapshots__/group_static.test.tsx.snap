// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`项目详情：纳入标准、排除标准 匹配快照 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div>
          <div>
            <p>
              纳入标准
            </p>
            <p
              data-testid="addJoin"
            >
              <span
                aria-label="plus-square"
                class="anticon anticon-plus-square"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  class=""
                  data-icon="plus-square"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M328 544h152v152c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V544h152c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8H544V328c0-4.4-3.6-8-8-8h-48c-4.4 0-8 3.6-8 8v152H328c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8z"
                  />
                  <path
                    d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-40 728H184V184h656v656z"
                  />
                </svg>
              </span>
               添加条件
            </p>
          </div>
          <div>
            <div
              class="ant-select ant-select-single ant-select-show-arrow"
              data-testid="selectJoin"
              style="width: 290px; height: 40px;"
            >
              <div
                class="ant-select-selector"
              >
                <span
                  class="ant-select-selection-search"
                >
                  <input
                    aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
                    aria-autocomplete="list"
                    aria-controls="rc_select_TEST_OR_SSR_list"
                    aria-haspopup="listbox"
                    aria-owns="rc_select_TEST_OR_SSR_list"
                    autocomplete="off"
                    class="ant-select-selection-search-input"
                    id="rc_select_TEST_OR_SSR"
                    readonly=""
                    role="combobox"
                    style="opacity: 0;"
                    unselectable="on"
                    value=""
                  />
                </span>
                <span
                  class="ant-select-selection-placeholder"
                >
                  请选择选项
                </span>
              </div>
              <span
                aria-hidden="true"
                class="ant-select-arrow"
                style="user-select: none;"
                unselectable="on"
              >
                <span
                  aria-label="down"
                  class="anticon anticon-down ant-select-suffix"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    class=""
                    data-icon="down"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                    />
                  </svg>
                </span>
              </span>
            </div>
            <span
              aria-label="delete"
              class="anticon anticon-delete"
              data-testid="delJoin"
              role="img"
              tabindex="-1"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="delete"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
                />
              </svg>
            </span>
          </div>
        </div>
        <div>
          <div>
            <p>
              排除标准
            </p>
            <p
              data-testid="addExclude"
            >
              <span
                aria-label="plus-square"
                class="anticon anticon-plus-square"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  class=""
                  data-icon="plus-square"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M328 544h152v152c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V544h152c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8H544V328c0-4.4-3.6-8-8-8h-48c-4.4 0-8 3.6-8 8v152H328c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8z"
                  />
                  <path
                    d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-40 728H184V184h656v656z"
                  />
                </svg>
              </span>
               添加条件
            </p>
          </div>
          <div>
            <div
              class="ant-select ant-select-single ant-select-show-arrow"
              data-testid="selectExclude"
              style="width: 290px; height: 40px;"
            >
              <div
                class="ant-select-selector"
              >
                <span
                  class="ant-select-selection-search"
                >
                  <input
                    aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
                    aria-autocomplete="list"
                    aria-controls="rc_select_TEST_OR_SSR_list"
                    aria-haspopup="listbox"
                    aria-owns="rc_select_TEST_OR_SSR_list"
                    autocomplete="off"
                    class="ant-select-selection-search-input"
                    id="rc_select_TEST_OR_SSR"
                    readonly=""
                    role="combobox"
                    style="opacity: 0;"
                    unselectable="on"
                    value=""
                  />
                </span>
                <span
                  class="ant-select-selection-placeholder"
                >
                  请选择选项
                </span>
              </div>
              <span
                aria-hidden="true"
                class="ant-select-arrow"
                style="user-select: none;"
                unselectable="on"
              >
                <span
                  aria-label="down"
                  class="anticon anticon-down ant-select-suffix"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    class=""
                    data-icon="down"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                    />
                  </svg>
                </span>
              </span>
            </div>
            <span
              aria-label="delete"
              class="anticon anticon-delete"
              data-testid="delExclude"
              role="img"
              tabindex="-1"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="delete"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
                />
              </svg>
            </span>
          </div>
        </div>
        <div>
          <button
            class="ant-btn ant-btn-primary"
            data-testid="submit"
            type="button"
          >
            <span>
              提 交
            </span>
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div>
        <div>
          <p>
            纳入标准
          </p>
          <p
            data-testid="addJoin"
          >
            <span
              aria-label="plus-square"
              class="anticon anticon-plus-square"
              role="img"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="plus-square"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M328 544h152v152c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V544h152c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8H544V328c0-4.4-3.6-8-8-8h-48c-4.4 0-8 3.6-8 8v152H328c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8z"
                />
                <path
                  d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-40 728H184V184h656v656z"
                />
              </svg>
            </span>
             添加条件
          </p>
        </div>
        <div>
          <div
            class="ant-select ant-select-single ant-select-show-arrow"
            data-testid="selectJoin"
            style="width: 290px; height: 40px;"
          >
            <div
              class="ant-select-selector"
            >
              <span
                class="ant-select-selection-search"
              >
                <input
                  aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
                  aria-autocomplete="list"
                  aria-controls="rc_select_TEST_OR_SSR_list"
                  aria-haspopup="listbox"
                  aria-owns="rc_select_TEST_OR_SSR_list"
                  autocomplete="off"
                  class="ant-select-selection-search-input"
                  id="rc_select_TEST_OR_SSR"
                  readonly=""
                  role="combobox"
                  style="opacity: 0;"
                  unselectable="on"
                  value=""
                />
              </span>
              <span
                class="ant-select-selection-placeholder"
              >
                请选择选项
              </span>
            </div>
            <span
              aria-hidden="true"
              class="ant-select-arrow"
              style="user-select: none;"
              unselectable="on"
            >
              <span
                aria-label="down"
                class="anticon anticon-down ant-select-suffix"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  class=""
                  data-icon="down"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                  />
                </svg>
              </span>
            </span>
          </div>
          <span
            aria-label="delete"
            class="anticon anticon-delete"
            data-testid="delJoin"
            role="img"
            tabindex="-1"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="delete"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
              />
            </svg>
          </span>
        </div>
      </div>
      <div>
        <div>
          <p>
            排除标准
          </p>
          <p
            data-testid="addExclude"
          >
            <span
              aria-label="plus-square"
              class="anticon anticon-plus-square"
              role="img"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="plus-square"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M328 544h152v152c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V544h152c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8H544V328c0-4.4-3.6-8-8-8h-48c-4.4 0-8 3.6-8 8v152H328c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8z"
                />
                <path
                  d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-40 728H184V184h656v656z"
                />
              </svg>
            </span>
             添加条件
          </p>
        </div>
        <div>
          <div
            class="ant-select ant-select-single ant-select-show-arrow"
            data-testid="selectExclude"
            style="width: 290px; height: 40px;"
          >
            <div
              class="ant-select-selector"
            >
              <span
                class="ant-select-selection-search"
              >
                <input
                  aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
                  aria-autocomplete="list"
                  aria-controls="rc_select_TEST_OR_SSR_list"
                  aria-haspopup="listbox"
                  aria-owns="rc_select_TEST_OR_SSR_list"
                  autocomplete="off"
                  class="ant-select-selection-search-input"
                  id="rc_select_TEST_OR_SSR"
                  readonly=""
                  role="combobox"
                  style="opacity: 0;"
                  unselectable="on"
                  value=""
                />
              </span>
              <span
                class="ant-select-selection-placeholder"
              >
                请选择选项
              </span>
            </div>
            <span
              aria-hidden="true"
              class="ant-select-arrow"
              style="user-select: none;"
              unselectable="on"
            >
              <span
                aria-label="down"
                class="anticon anticon-down ant-select-suffix"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  class=""
                  data-icon="down"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                  />
                </svg>
              </span>
            </span>
          </div>
          <span
            aria-label="delete"
            class="anticon anticon-delete"
            data-testid="delExclude"
            role="img"
            tabindex="-1"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="delete"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
              />
            </svg>
          </span>
        </div>
      </div>
      <div>
        <button
          class="ant-btn ant-btn-primary"
          data-testid="submit"
          type="button"
        >
          <span>
            提 交
          </span>
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
